spec/abstractops.html

```diff
@@ -196,31 +196,48 @@ <h1>ToShowOffsetOption ( _normalizedOptions_ )</h1>
     <h1>
       ToTemporalRoundingIncrement (
         _normalizedOptions_: an Object,
-        _dividend_: an integer or *undefined*,
+      ): either a normal completion containing a Number, or an abrupt completion
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>
+        It extracts the value of the property named *"roundingIncrement"* from _normalizedOptions_, makes sure it is a finite Number of at least 1, and returns that value.
+        It performs no further validation.
+      </dd>
+    </dl>
+    <emu-alg>
+      1. Let _increment_ be ? GetOption(_normalizedOptions_, *"roundingIncrement"*, *"number"*, *undefined*, *1*<sub>𝔽</sub>).
+      1. If _increment_ is *NaN* or *+&infin;*<sub>𝔽</sub>, throw a *RangeError* exception.
```

Comment:
I know the next step handles -Infinity, but it would be more clear IMO to handle it here with the other special values (and [GetOption](https://tc39.es/proposal-temporal/#sec-getoption) never returns NaN anyway).
```suggestion
      1. If _increment_ is not finite, throw a *RangeError* exception.
```